app/api/contacts/search/route.ts

```diff
@@ -25,5 +29,15 @@ export async function GET(request: NextRequest) {
     .where(whereClause)
     .limit(20);
 
-  return NextResponse.json(contacts);
+  return contacts;
 }
+
+export const GET = withApiWrapper(searchContactsHandler, {
+  getBody: async (request: Request) => {
```

Comment:
We shouldn't need a custom getBody for GET requests. We should know to use query parameters when it's a get request